docs/dasharo-tools-suite/documentation.md

```diff
@@ -5,18 +5,18 @@
 Dasharo Tools Suite was prepared to run on x86 platforms, but we can confirm
 that it boots on the following platforms:
 
-* ASUS KGPE-D16
-* Dell OptiPlex 7010/9010
+* ASUS KGPE-D16,
+* Dell OptiPlex 7010/9010,
 * MSI PRO Z690-A DDR4 ([test
-  report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75))
+  report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75)),
 * NovaCustom NV4x ([test
-  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=536764189&range=A161))
+  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=536764189&range=A161)),
 * NovaCustom NS5x/7x ([test
-  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675&range=A174))
-* Protectli FW6/VP46xx
+  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675&range=A174)),
+* Protectli FW6/VP46xx.
```

Comment:
Are we sure, that DTS might be used on Protectli platforms? And if so, why do we describe two platform families in one section?

Reply:
@sulewskiprzemyslaw probably this is an info that I get from TAT on v1.1.0 release. Please remember that DTS is more than zero-touch initial deploy. Fact is that we cannot do that on Protectli platfroms but still we can boot DTS and e.g. dump HCL Report, or use installed on DTS flashrom.

Reply:
@TomaszAIR OK, so if we declared, that DTS might work on Protectli platforms, we should:
1. Expand the test matrix dedicated to these platforms. 
2. Check if the features, you described, work correctly on these platforms.

I would not want to declare compatibility with a particular platform without testing.

Reply:
@sulewskiprzemyslaw don't we use DTS on Protectli platforms? Plus does this sentence `Dasharo Tools Suite was prepared to run on x86 platforms, but we can confirm that it boots on the following platforms:`  (which is at the beginning of this section)  not explain that we only mention about DTS booting? No one talk about running any features or tests.

This issue https://github.com/Dasharo/dasharo-issues/issues/338 should resolve problem about `Supported hardware` section and we could just post here results from every device.

Reply:
@TomaszAIR So far, DTS has not been tested on these platforms.

Reply:
@TomaszAIR remove information about Protectli (commit: https://github.com/Dasharo/docs/pull/369/commits/015ed617dfa60cb39d66268e40268c226aa76d7c)